homeassistant/components/camera/__init__.py

```diff
@@ -930,6 +965,28 @@ async def handle(self, request: web.Request, camera: Camera) -> web.StreamRespon
             raise web.HTTPBadRequest from err
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "camera/capabilities",
+        vol.Required("entity_id"): cv.entity_id,
+    }
+)
+@websocket_api.async_response
+async def ws_camera_capabilities(
+    hass: HomeAssistant, connection: ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Handle get camera capabilities websocket command.
+
+    Async friendly.
+    """
+    try:
+        camera = get_camera_from_entity_id(hass, msg["entity_id"])
```

Comment:
In the WebRTC commands we didn't catch the `HomeAssistantError` coming from `get_camera_from_entity_id` since it will be handled by the common exception handler. I think we should be consistent.